brave/browser-laptop
# Refactor of tabs related components (redux component)

Resolves #8690

Test Plan:
- try reordering 4 tabs with dnd
- try reordering 4 pinned tabs with dnd 
- should pass regular tab manual test https://github.com/brave/browser-laptop/wiki/Manual-Tests#tabs-pinning-and-tear-off-tabs
